static/src/stylesheets/head.hosted-amp.scss

```diff
@@ -12,6 +12,8 @@
 @import 'amp/_type';
 @import 'amp/_fonts';
 @import 'amp/_helpers';
+@import 'amp/_content';
```

Comment:
This one is a really big file. Most of this I don't think hosted pages use. Or, they overwrite. Are you sure you need this file?